using-valtimo/zgw/documents/zgw-documents.md

```diff
@@ -40,4 +40,49 @@ The contents should follow the structure in the example below:
 }
 ```
 
+### List columns
+List columns can be configured to change what data is shown in the document list view. 
+
+#### Autodeployment
+List column configuration can also be loaded via autodeployment.
+To do so, create a json file in the application classpath which conforms to the following pattern: `*.zgw-document-list-column.json`
+
+The contents should follow the structure in the example below:
+
+**my-zaak.zgw-document-trefwoorden.json**
+```json
+{
+  "changesetId": "my-zaak.zgw-document-list-columns-v1",
+  "case-definitions": [
+    {
+      "key": "my-zaak",
+      "columns": [
+        "AUTEUR",
```

Comment:
This format has changed. It now is a list of objects instead of a list of strings